src/chigame/api/tests/factories.py

```diff
@@ -106,3 +106,18 @@ class Meta:
         model = Chat
 
     tournament = SubFactory(TournamentFactory)
+
+
+class LobbyFactory(DjangoModelFactory):
+    class Meta:
+        model = Lobby
+
+    match_status = Lobby.Lobbied
+    name = Faker("word")
+    game = SubFactory(GameFactory)
+    game_mod_status = Lobby.Default_game
+    created_by = SubFactory(UserFactory)
+    min_players = 2
+    max_players = 4
```

Comment:
When converting `min_players` to a `LazyAttribute`, change this to another `LazyAttribute` which uses the values of `min_players` such that there will never be a sitation where `min_players > max_players`